code/style.css

```diff
@@ -1,58 +1,250 @@
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family:'Permanent Marker', cursive;
+  background-color: #3D915D;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  background-image: url(https://i.pinimg.com/564x/e2/ae/f6/e2aef657b2faac87de8c27ee392bc5ca.jpg);
+}
+
+h1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'Permanent Marker';
+  background: white;
+  flex-direction: column;
+  width: 100%;
+  font-size: 100px;
 }
 
 p {
-  margin: 1px;
-  font-weight: 300;
+  font: 'Permanent Marker';
   background-color: #fff;
+  font-size: 40px;
+  font-weight: 300;
+  margin: 1px;
 }
 
 .cards {
-  margin: 50px auto;
-  width: 90%;
-  height: 600px;
-  background: #ccc;
+  flex-wrap: wrap;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  cursor: pointer;
+  margin: 50px auto;
+  border-radius: 25px;
+  width: 80%;
+  height: 70%;
+  background: rgb(10, 53, 10);
+}
+
+#weaponName, #killerName, #roomName {
+  font-size: 18px;
 }
 
 .killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+  flex-direction: row;
+  background: #916563; 
   text-align: center;
+  box-shadow: 5px 10px 8px #888888;
   padding-top: 30px;
   border-radius: 16px;
+  width: 250px;
+  height: 400px;
+}
+
+.killer-deck:hover, .weapon-deck:hover, .room-deck:hover {
+  background:#9A2931;
 }
 
 .weapon-deck {
```

Comment:
There is a lot of shared styles between killer-deck, weapon-deck and room-deck - They could be unified to one class, and have separate classes only for styles that are not shared between them